meson.build

```diff
@@ -1763,25 +1810,24 @@ dtrace_command = [
 ]
 dtrace_deps = []
 
-if host_os == 'freebsd' and cpu == 'aarch64'
-    with_dtrace = false
-    message('The version of dtrace in aarch64 FreeBSD is currently unsupported')
-elif not get_option('with-dtrace')
-    with_dtrace = false
+if not enable_dtrace
+    have_dtrace = false
+elif host_os == 'freebsd' and cpu == 'aarch64'
+    have_dtrace = false
+    warning('The version of dtrace in aarch64 FreeBSD is currently unsupported')
```

Comment:
How about we leave a reference to https://wiki.freebsd.org/arm64#Known_Limitations in either a code comment, or the message?

Reply:
Done